src/v/raft/heartbeat_manager.cc

```diff
@@ -48,7 +47,7 @@ struct heartbeat_requests {
 
 static heartbeat_requests requests_for_range(
   const consensus_set& c, clock_type::duration heartbeat_interval) {
-    absl::flat_hash_map<
+    absl::btree_map<
```

Comment:
title of the commit got out of sync with the change (btree map not node hash map!)